### ISSUE TITLE
URL 404 in the upgrade language file

### ISSUE THREAD

**Issue by horus68** [salesagility/SuiteCRM#1097]
URL 404 in the upgrade language file (line 61)
\scripts\en_us.lang.php

`'ERR_PERFORM_NO_TCPIP'             => '<b>We could not detect an internet connection.</b> When you do have a connection, please visit <a href=\"http://www.SuiteCRM.com/home/index.php?option=com_extended_registration&task=register\">http://www.SuiteCRM.com/home/index.php?option=com_extended_registration&task=register</a> to register with SuiteCRM. By letting us know a little bit about how your company plans to use SuiteCRM, we can ensure we are always delivering the right application for your business needs.',`

I think it would also be useful to clean up that file and include translations (already translated via Crowdin if you want) but there is no place to handle those contributions

**Comment by horus68** [comment]
@mattlorimer Its time to update that file (this is tagged as a suggestion but it needs to be considered as a Bug)!
There is no Github available for this file so here it goes:

Line 32
Not:
`	'ERR_DB_EXISTS_WITH_CONFIG'		=> 'Database already exists with config data.  To run an install with the chosen database, please re-run the install and choose: "Drop and recreate existing SuiteCRM tables?"  To upgrade, use the Upgrade Wizard in the Admin Console.  Please read the upgrade documentation located <a href="http://www.sugarforge.org/content/downloads/" target="_new">here</a>.',`

But this
`	'ERR_DB_EXISTS_WITH_CONFIG'		=> 'Database already exists with config data.  To run an install with the chosen database, please re-run the install and choose: "Drop and recreate existing SuiteCRM tables?"  To upgrade, use the Upgrade Wizard in the Admin Console.  Please read the upgrade documentation located <a href="http://www.suitecrm.com" target="_new">here</a>.',`

Line 61
Not:
`	'ERR_PERFORM_NO_TCPIP'				=> '<b>We could not detect an internet connection.</b> When you do have a connection, please visit <a href=\"http://www.SuiteCRM.com/home/index.php?option=com_extended_registration&task=register\">http://www.SuiteCRM.com/home/index.php?option=com_extended_registration&task=register</a> to register with SuiteCRM. By letting us know a little bit about how your company plans to use SuiteCRM, we can ensure we are always delivering the right application for your business needs.',`

But:
`	'ERR_PERFORM_NO_TCPIP'				=> '<b>We could not detect an internet connection.</b> When you do have a connection, please visit <a href=\"http://www.suitecrm.com\">http://www.suitecrm.com</a> to register with SuiteCRM. By letting us know a little bit about how your company plans to use SuiteCRM, we can ensure we are always delivering the right application for your business needs.',`

Line 114
Not:
`	'LBL_DBCONF_PRIV_USER_DIRECTIONS'	=> 'This privileged database user must have the proper permissions tocreate a database, drop/create tables, and create a user.  This privileged database user will only be used to perform these tasks as needed during the installation process.  You may also use the same database user as above if that user has sufficient privileges.',`

But
`	'LBL_DBCONF_PRIV_USER_DIRECTIONS'	=> 'This privileged database user must have the proper permissions to create a database, drop/create tables, and create a user.  This privileged database user will only be used to perform these tasks as needed during the installation process.  You may also use the same database user as above if that user has sufficient privileges.',`

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [URL 404 in the upgrade language file](https://trello.com/c/Wp9pHfBy/113-url-404-in-the-upgrade-language-file)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
